File: src/main/java/org/dedda/bratwurst/parse/CharToIntParser.java
package org.dedda.bratwurst.parse;

import org.dedda.bratwurst.lang.CharToInt;

/**
 * Created by dedda on 1/28/16.
 *
 * @author dedda
 */
public class CharToIntParser extends ExpressionParser {

    @Override
    public CharToInt parse(String data, int lineNumber) {
        String varname = data.split(" ")[1];
        return new CharToInt(lineNumber, varname);
    }
}

File: src/main/java/org/dedda/bratwurst/parse/CharToIntParser.kt
package org.dedda.bratwurst.parse

import org.dedda.bratwurst.lang.CharToInt

/**
 * Created by dedda on 1/28/16.
 *
 * @author dedda
 */
class CharToIntParser : ExpressionParser() {
    override fun parse(data: String, lineNumber: Int): CharToInt {
        val varname = data.split(" ".toRegex()).toTypedArray()[1]
        return CharToInt(lineNumber, varname)
    }
}